This file is part of MXE.
See index.html for further information.

This patch was taken from:
https://sourceforge.net/p/mingw-w64/code/6479/

--- a/mingw-w64-headers/crt/math.h
+++ b/mingw-w64-headers/crt/math.h
@@ -36,7 +36,7 @@
 #endif
 #endif
 
-#ifndef __STRICT_ANSI__
+#if !defined(__STRICT_ANSI__) || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) || defined(_BSD_SOURCE) 
 #define M_E		2.7182818284590452354
 #define M_LOG2E		1.4426950408889634074
 #define M_LOG10E	0.43429448190325182765

